src/KubernetesClient/CertUtils.cs

```diff
@@ -19,22 +18,32 @@ public static class CertUtils
         /// Load pem encoded cert file
         /// </summary>
         /// <param name="file">Path to pem encoded cert file</param>
-        /// <returns>x509 instance.</returns>
-        public static X509Certificate2 LoadPemFileCert(string file)
+        /// <returns>List of x509 instances.</returns>
+        public static IList<X509Certificate2> LoadPemFileCert(string file)
         {
-            var certs = new X509CertificateParser().ReadCertificates(File.OpenRead(file));
-            var store = new Pkcs12StoreBuilder().Build();
-            foreach (X509Certificate cert in certs)
```

Comment:
here, already read all certs in the file.
the reason did not return a collection is due to the the return type is `X509Certificate2` before.

Reply:
Added this back